gradle-consistent-versions-idea-plugin/src/main/java/com/palantir/gradle/versions/intellij/VersionPropsFileListener.java

```diff
@@ -62,24 +64,81 @@ public ChangeApplier prepareChange(List<? extends VFileEvent> events) {
                 .filter(Project::isInitialized)
                 .filter(Predicate.not(ComponentManager::isDisposed))
                 .filter(project -> versionPropsEvents.stream()
-                        .anyMatch(event -> event.getPath().startsWith(project.getBasePath())))
+                        .anyMatch(event -> event.getPath().startsWith(project.getBasePath())
+                                && !isFileMalformed(project, event.getFile())))
                 .toList();
 
         return new ChangeApplier() {
             @Override
             public void afterVfsChange() {
+                String taskName = "writeVersionsLock";
```

Comment:
nit: I think you can move this to a static field

Reply:
awesome have made it static - thanks for the review!